src/components/interface/Header/NavItem.d.ts

```diff
@@ -8,6 +8,7 @@ export interface NavItemProps {
   link?: string;
   current?: boolean;
   asLink?: React.ReactElement<any>;
+  onClick?: (...args: any[])=>any;
```

Comment:
can you properly type it with proper event type (I know it's not the case right now, but I will be nice to tend to that)

Reply:
I copy the `onClick` type from src/components/interface/Header/ToolItem.d.ts 

I think we could use `  onClick?: (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>)=>any;` in `Link.d` and inherits it in `NavItem`/`ToolItem` ?

Reply:
https://github.com/dataesr/react-dsfr/pull/312/commits/9f2e38dc88bad6f87c9b7437d1710d9368b4ac4a make `onClick` depends on `Link::onClick` type. Is it good for you ?

Reply:
I don't think we should make them depend from each other as they this is an implementation choice that can change.
but using `onClick?: (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>)=>any;`is definitely a good idea :)